.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
```

Comment:
Actions is throwing some warnings that this will be updated from `18.04` to `20.04` soon. Good to keep in mind, but I don't think it's a huge risk here.